README.md

```diff
@@ -134,22 +143,25 @@ func (us *UserStorageData) Get(userID, campaignKey string) schema.UserData {
     /*
     // UserData  struct
     type UserData struct {
-        UserID        string
-        CampaignKey   string
-        VariationName string
-    }
+		UserID         string
```

Comment:
indentation issue? or GitHub rendering?

Reply:
Looks fine in VsCode when i push, but then goes out of the indentation. Looks like a gitHub issue.

Reply:
is it because of mixing spaces and tabs? still doesn't look good

Reply:
issue is also visible while previewing file - https://github.com/decabits/vwo-golang-sdk/blob/ef47221539a3dbc3fde746d8072108a08cf10c67/README.md

Reply:
![image](https://user-images.githubusercontent.com/40268761/89521806-ccbf2c80-d7fd-11ea-9649-77ba38a4625d.png)
![image](https://user-images.githubusercontent.com/40268761/89521842-dba5df00-d7fd-11ea-9608-9b0d0cf98355.png)
Looks fine.

Reply:
okay...something weird at my end